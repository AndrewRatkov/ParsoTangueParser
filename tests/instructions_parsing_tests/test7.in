h(2, "hrundelair");