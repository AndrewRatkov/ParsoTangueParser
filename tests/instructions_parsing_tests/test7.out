[CALL VOID h]------->[INT,2]
      |
      +------------->[STR,"hrundelair"]